### PR TITLE
Adds reset; fixes multiple target issue.

### LINKED ISSUE #126: Only last defined Image Target recognized when multiple Image Targets defined?
Hey 👋 ,

I was following along the simple demo example, but had some issues getting multiple target images to work with the kit. Here's an example commit with an additional image target defined in the simple demo with some details on what I was encountering in the commit description:

https://github.com/adrianbautista/perception-toolkit/commit/ad7e4c37f61a9ffb963b316ad9cc4bf761b5696c

> ...When first starting the simple demo, kit only seems to store only instances of the last image target (*rocks*). But if I stop the kit and start it again, it will say two image targets stored
(*lighthouse*, *rocks*). However when showing the lighthouse image to the camera the kit recognizes *rocks* before *lighthouse* thereby rendering the *rocks* card instead.

And some console screenshots:

first-start

*simple demo first load*

---

second-start

*simple demo reloaded*

---

redundant-detection

*simple demo reloaded, attempt at recognizing the first Image Target defined in artifactSources*

So wanted to double check if I'm misunderstanding how planar tracking is used for the image targets. I'm still debugging this, but let me know if any additional information/context could be helpful!

### PR DESCRIPTION
**Pull request by paullewis**
Fixes #126

### CODE REVIEW

File: src/detectors/planar-image/planar-image.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ import { Marker } from '../../../defs/marker.js';
 import { DEBUG_LEVEL, log } from '../../utils/logger.js';
 
 class Detector {
-  private readonly targets = new Map<number, DetectableImage>();
+  private readonly targets = new Map<number, { id: string }>();
```
Comment (mmocny) [inline src/detectors/planar-image/planar-image.ts]: `DetectedImage` has this interface, if you want.

Comment (paullewis) [reply src/detectors/planar-image/planar-image.ts]: We ping the whole thing over to the worker and back, which means we'd have to transfer the whole DetectableImage :-/

Comment (mmocny) [reply src/detectors/planar-image/planar-image.ts]: I said `Detec*ted*Image` not `Detec*table*Image` ;)

The difference is the lack fo `media:`

Comment (paullewis) [reply src/detectors/planar-image/planar-image.ts]: Oops

**Pull request merged**